Services/UserService/Controllers/UserController.cs

```diff
@@ -1,39 +1,53 @@
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+using Common.Enums;
+using Common.Exceptions;
 using Common.Models.User;
-using Microsoft.AspNetCore.Http;
+using Common.Response;
 using Microsoft.AspNetCore.Mvc;
-using Supabase;
+using UserService.Service;
 
-namespace UsersService.Controllers
+namespace UserService.Controllers
 {
-    [Route("api/v1")]
+    [Route("api/user")]
     [ApiController]
     public class UserController : ControllerBase
```

Comment:
Why do I not see any Usergroup conrtoller? Have you decided to create a standalone service for usergroup? If so, that is ok but Task in Jira needs to be created in order for us both to know what is going on.